ignite/cmd/network_request_reject.go

```diff
@@ -16,8 +16,14 @@ func NewNetworkRequestReject() *cobra.Command {
 		Use:     "reject [launch-id] [number<,...>]",
 		Aliases: []string{"accept"},
 		Short:   "Reject requests",
-		RunE:    networkRequestRejectHandler,
-		Args:    cobra.ExactArgs(2),
+		Long: `The "reject" command is used by a chain's coordinator to approve requests.
```

Comment:
I am not sure if "to **refuse** requests" might be better than "reject" to remove repetition. Maybe that would be very nit-picky :)

```suggestion
		Long: `The "reject" command is used by a chain's coordinator to reject requests.
```

Reply:
Makes sense 👍